lib/main.dart

```diff
@@ -1,9 +1,16 @@
 import 'package:flutter/material.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
 
 import 'HomePage.dart';
 
-void main() {
-  runApp(MyApp());
+Future<void> main() async {
+  await SentryFlutter.init(
+    (options) {
+      options.dsn = 'https://example@sentry.io/add-your-dsn-here';
```

Comment:
Need to find a way to add our internal Sentry URL, without publishing the URL + DSN on a public repo

Reply:
This way I solved this:

* I added `.github/secret_constants.ci.dart` that contains fake-secrets
* The actual secrets file is in the `.gitignore` because we never want this file to be public
* CI copies that file into the correct location: `lib/secret_constants.dart`
* You both would have to do the same: `cp .github/secret_constants.ci.dart lib/secret_constants.dart` and then fill in the _actual_ secrets. (I would send you the correct DSN via slack)